### PR TITLE
New aci

### PR DESCRIPTION
**Pull request by rvirding**
Add an ACI to the HTTP compiler interface.

[PT-163022973](https://www.pivotaltracker.com/story/show/163022973)

### CODE REVIEW

File: config/swagger.yaml [hunk 1]
```diff
@@ -134,6 +134,16 @@ definitions:
         type: string
     required:
       - reason
+  ACIJsonData:
+    type: object
+    properties:
+      decoded_aci:
+        type: string
```
Comment (hanssv) [inline config/swagger.yaml]: Is it a string?! Didn't `aeso_aci` return a JSON object?

File: rebar.config [hunk 2]
```diff
@@ -10,7 +10,7 @@
         {aeserialization, {git, "https://github.com/aeternity/aeserialization.git",
                            {ref, "179dd41"}}},
         {aesophia, {git, "https://github.com/aeternity/aesophia.git",
-                    {ref, "e94c1f9"}}},
+                    {ref, "cdc7b90"}}},
```
Comment (hanssv) [inline rebar.config]: This is the commit that is already on master?! Is this not rebased?

File: config/swagger.yaml [hunk 3]
```diff
@@ -137,10 +137,13 @@ definitions:
   ACI:
     type: object
     properties:
-      aci:
-        $ref: '#/definitions/EncodedByteArray'
+      encoded_aci:
```
Comment (hanssv) [inline config/swagger.yaml]: In what sense is it encoded?

File: rebar.config (hunk 2 above)
Comment (rvirding) [reply rebar.config]: Shouldn't it be pointing to aesophia master?

Comment (hanssv) [reply rebar.config]: Ok, it is easier to review if you rebase while you had to do changes anyway. But nevermind.

**Pull request merged**